internal/data_source_huaweicloud_availability_zones.go

```diff
@@ -0,0 +1,73 @@
+package internal
+
+import (
+	"sort"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+
+	"github.com/chnsz/golangsdk/openstack/compute/v2/extensions/availabilityzones"
```

Comment:
should have a break line here for different packages.

Reply:
removed